challenges/objects-arrays.js

```diff
@@ -61,10 +92,22 @@ The resulting contact information strings should have a space between the first
 
 Log the result of your new array. */
 const contactInfo = [];
+
+for (count = 0; count <graduates.length; count++) {
```

Comment:
i would stick with using i vs other variables (unless your nesting 4 loops) while it works dev's will be looking for the i  when looping